lib/send-html.js

```diff
@@ -0,0 +1,35 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var ejs = require('ejs');
+var fs = require('fs');
+var path = require('path');
+
+module.exports = sendHTML;
+
+function sendHTML(res, data, options) {
+  var assetDir = path.resolve(__dirname, '../views/');
+  var views = {
+    default: path.resolve(assetDir, 'default-error.ejs'),
+    404: path.resolve(assetDir, assetDir, '404.ejs'),
+  };
+  //TODO: Let user specify their own Views
+  var view = views.default;
+
+  // using specific error template
+  if (views.hasOwnProperty(data.statusCode)) {
+    view = views[data.statusCode];
+  }
+
+  render(res, data, view);
+};
+
+function render(res, data, view) {
+  res.setHeader('Content-Type', 'text/html; charset=utf-8');
+  var toRender = {options: {}, data: data};
+  ejs.renderFile(view, toRender, {}, function(err, body) {
```

Comment:
I personally think it's best to compile the templates at the time when `send-html.js` is loaded via `fs.readFileSync` + `ejs.compile`.